src/main/resources/META-INF/archetype-post-generate.groovy

```diff
@@ -169,9 +169,15 @@ if (includeCommerce == "n") {
     }
 }
 
-// if forms flag is not set, forms specific content/configuration/proxy components should be deleted
+// if forms flag is not set, forms specific components, template-types, templates, themes should be deleted
 if (includeForms == "n") {
     assert new File("$appsFolder/components/aemformscontainer").deleteDir()
+    assert new File("$confFolder/settings/wcm/template-types/af-page").deleteDir()
+    assert new File("$confFolder/settings/wcm/templates/basic-af").deleteDir()
+    assert new File("$confFolder/settings/wcm/templates/blank-af").deleteDir()
+    assert new File("$uiContentPackage/src/main/content/jcr_root/content/dam/formsanddocuments-themes").deleteDir()
+    assert new File("$uiContentPackage/src/main/content/jcr_root/content/dam/$appId/sample_logo.png").deleteDir()
```

Comment:
again its aligned to how all assets are added for different products, these are not namespaced, do you still prefer namespacing?

Reply:
> how all assets are added for different products, these are not namespaced

I would prefer name-spacing for better management among add-ons.

Reply:
After the change in https://github.com/adobe/aem-project-archetype/pull/647 this assets would not get deployed because of the new filter definition.
Can you move the assets to the formsanddocuments-themes?

Reply:
formsanddocuments-themes is the path where all the form themes reside. @bpauli PR you are referring to moved the root path to structure, and is still being used to add another image assets.jpg. I don't understand how will additional images fail to be added to root folder.
Are you saying these paths would be conditionally generated? Aren't these always generated?

Reply:
updated PR with suggested filter changes.